pom.xml

```
@@ -240,19 +240,17 @@ flexible messaging model and an intuitive client API.</description>
     <jclouds.version>2.6.0</jclouds.version>
     <guice.version>5.1.0</guice.version>
     <sqlite-jdbc.version>3.47.1.0</sqlite-jdbc.version>
-    <mysql-jdbc.version>8.0.11</mysql-jdbc.version>
-    <postgresql-jdbc.version>42.5.5</postgresql-jdbc.version>
+    <mysql-jdbc.version>9.1.0</mysql-jdbc.version>
+    <postgresql-jdbc.version>42.7.7</postgresql-jdbc.version>
     <clickhouse-jdbc.version>0.4.6</clickhouse-jdbc.version>
-    <mariadb-jdbc.version>2.7.5</mariadb-jdbc.version>
+    <mariadb-jdbc.version>3.5.3</mariadb-jdbc.version>
     <openmldb-jdbc.version>0.4.4-hotfix1</openmldb-jdbc.version>
     <json-smart.version>2.5.2</json-smart.version>
     <opensearch.version>2.16.0</opensearch.version>
     <elasticsearch-java.version>8.15.3</elasticsearch-java.version>
-    <debezium.version>1.9.7.Final</debezium.version>
-    <debezium.postgresql.version>42.5.5</debezium.postgresql.version>
-    <debezium.mysql.version>8.0.33</debezium.mysql.version>
-    <!-- Override version that brings CVE-2022-3143 with debezium -->
-    <wildfly-elytron.version>1.15.16.Final</wildfly-elytron.version>
+    <debezium.version>3.2.2.Final</debezium.version>
+    <debezium.postgresql.version>42.7.7</debezium.postgresql.version>
+    <debezium.mysql.version>9.1.0</debezium.mysql.version>
```

The latest version is 9.4.0 at maven central, https://central.sonatype.com/artifact/com.mysql/mysql-connector-j

```suggestion
    <debezium.mysql.version>9.4.0</debezium.mysql.version>
```

In addition, there's a need to update the groupId and artifactId in dependencyManagement in pulsar-io/debezium/mysql/pom.xml

The groupId and artifactId has changed in 9.x:
https://dev.mysql.com/doc/connector-j/en/connector-j-installing-maven.html

mysql -> com.mysql
mysql-connector-java -> mysql-connector-j

This would be needed in in pulsar-io/debezium/mysql/pom.xml for this to be effective
```xml
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>com.mysql</groupId>
        <artifactId>mysql-connector-j</artifactId>
        <version>${debezium.mysql.version}</version>
      </dependency>
    </dependencies>
  </dependencyManagement>
```

---

Good catch. Updated!